projects/tn-ra-manual.md

```diff
@@ -4,3 +4,4 @@ Notizen zu manuellen Anpassungen
 - Bedingungen mit `nummerDerBahnstrecke` angepasst (zu `nummerderbahnstrecke`)
 - Anpassung von Groovy Typ-Relationen mit korrigierten Typ- und Eigenschafts-Namen
 - Explizite Behandlung von Codes als Strings in Groovy Typ-Relationen (um Logik kompatibel zu halten)
+- Datumsformat von RLP `yyyy-MM-dd'T'HH:mm:ss` formatiert als String wird explizit in der custom function `adv.inspire.date.earliest` bei der Umwandlung in ein Datums-Objekt berücksichtigt.
```

Comment:
Cleaner solution would be to have that change in a separate `docs` commit IMO but no need to change the PR because of it.

Reply:
I meant a separate commit containing only this change on the documentation. But as I mentioned, no need for adaptaions.